docs/2-migration/07-claiming-zen.md

```diff
@@ -87,5 +88,120 @@ After a successful  claim the following event will be emitted:
 event Claimed(address destAddress, bytes20 zenAddress, uint256 amount)
 ```
 
+### Direct Claim - 1st method
 
+This method allows a direct claim of special UTXOs generated deterministically from a BaseAddress.<br/>
+This is a special usecase for users that can't generate a signed message, and requires they create this special UTXO in the old mainchain before the migration.
 
+This method can be invoked by anyone (not mandatory the sender of the claim tx  to be the same destination address).
+
+The Solidity method to execute this claim is the following:
+
+```
+    function claimDirect(address baseDestAddress) public 
+```
+
+#### Parameters details:
+
+- baseDestAddress: Destination address on Base of the funds to be claimed.<br/>
+
+Any Base address is a valid *baseDestAddress*, and that address could claim the funds for the Zend Address generated as such:
```

Comment:
I know it is obvious, but maybe it can be added that the destaddress should have a private key

Reply:
you said: is obious :-)